lang/ilias_en.lang

```diff
@@ -17089,11 +17089,11 @@ trac#:#user_total#:#User Total
 trac#:#view_mode#:#View Mode
 tstv#:#tstv_create#:#Create Test Certificate
 tstv#:#tstv_create_info#:#Select a completed test to generate a certificate for it.
-ui#:#datatable_multiactionmodal_actionlabel#:#operation
```

Comment:
I assume this label appears at the beginning of a line and is not part of a constructed term. In this case it should be ‘Operation‘

Reply:
This is already replaced with `Action for All Entries` below.